pom.xml

```diff
@@ -138,6 +142,12 @@
         </dependency>
 
         <dependency>
+            <groupId>javax.mail</groupId>
```

Comment:
Interesting...does this happen on master branch now too, before this PR?  Or is it something new added in this PR?